input/papers/2016/2016arXiv160908671H/tev-000095.yaml~

```diff
@@ -0,0 +1,18 @@
+paper_id: 2006A&A...457..899A
```

Comment:
This file `input/papers/2016/2016arXiv160908671H/tev-000095.yaml~` is a backup save file from your editor.
It doesn't belong in the repository, please remove it.

You could add a line
```
*~
```
anywhere in
https://github.com/gammapy/gamma-cat/blob/master/.gitignore
to tell git to ignore these files.

See https://git-scm.com/docs/gitignore for further info how it works.